### ISSUE TITLE
Create reaction gifs public repository site

### ISSUE THREAD

**Issue by shorttompkins** [shorttompkins/mrw.lol#1]
Lets take the repo and expand on it by creating a real-world application.  Lets build apon imgPloadr by creating a website that allows users to register/login (using social network authentication i.e. Twitter/Facebook) and upload and tag a collection of gifs. Users can also browser other users gifs (by tag) and like them to add a copy to their own collection.

**Comment by shaunparris** [comment]
@shorttompkins 
Did you decide to not run with this idea? I would love to have an app like this. I have been brainstorming how to do it on my own, but don't have the dev skills. I only have some basic scripting skills..

**Comment by shorttompkins** [comment]
@shaunparris yep still going - I guess it looks weird that I closed this issue.  Keep an eye on the [README](https://github.com/shorttompkins/mrw.lol/blob/master/README.md) and the [Changelog](https://github.com/shorttompkins/mrw.lol/blob/master/changelog.md) files ;)

**Comment by shorttompkins** [comment]
Reopening as it looked odd to close this.

**Comment by shaunparris** [comment]
Awesome! I look forward to it. I come from a QA background. Let me know if
you would like some Alpha or Beta testing.
